packages/material-ui-styles/src/makeStyles/makeStyles.js

```diff
@@ -242,7 +242,7 @@ export default function makeStyles(stylesOrCreator, options = {}) {
       React.useDebugValue(classes);
     }
     if (process.env.NODE_ENV !== 'production') {
-      const allowlistedComponents = ['MuiButton', 'MuiTypography'];
+      const allowlistedComponents = ['MuiButton', 'MuiTypography', 'MuiAvatar'];
```

Comment:
Sorry, I thought we weren't going to change this given that:
1. It will eventually be removed,
2. whitelist & blacklist have no racial connotations or etymology,
3. It's slacktivism.

Anyway, since it's been modified, let's at least have it at least make sense:

```suggestion
      const allowedComponents = ['MuiButton', 'MuiTypography', 'MuiAvatar'];
```
```suggestion
      const supportedComponents = ['MuiButton', 'MuiTypography', 'MuiAvatar'];
```

Reply:
I'll change it to `supportedComponents`